static/app/components/draggableTabs/draggableTabList.tsx

```diff
@@ -230,7 +230,6 @@ function Tabs({
               dragConstraints={dragConstraints} // dragConstraints are the bounds that the tab can be dragged within
               dragElastic={0} // Prevents the tab from being dragged outside of the dragConstraints (w/o this you can drag it outside but it'll spring back)
               dragTransition={{bounceStiffness: 400, bounceDamping: 40}} // Recovers spring behavior thats lost when using dragElastic=0
-              transition={{delay: -0.1}} // Skips the first few frames of the animation that make the tab appear to shrink before growing
```

Comment:
This doesn't seem like a stable fix. We're assuming that the jitter happens in the first 100ms. The problem may return if say we made the animation longer, or changed the sequencing in some way.

Reply:
@vuluongj20  This change is unrelated to the jitter - this delay was added when the animation still stretched the text within the tab. It's purpose was to skip the initial, sloppy frames of the animation when the text appeared to shrink before expanding to it's full width. 

Now that only the tab text's position animates and not the size, this change is unnecessary. I just forgot to remove it when I changed the animation initially.

Reply:
The jitter orginated within `editableTabTitle.tsx`, and was caused by the fact that the tab title switched from a `<div>` to an `<input>` when _switching_ tabs. The real fix comes from making sure the tab title becomes an `<input>` only when the tab is actively being edited, rather than just being selected.